### DISCUSSION TITLE
Joint vs Separate Search Heterogeneous Samples

### DISCUSSION THREAD

**Discussion by abadgerw** [vdemichev/DiaNN#1107] [Q&A]
Apologies for the likely naive question: We have captured areas of grey matter and white matter and performed downstream DIA proteomics at the same time in one run and I wanted to get some thoughts/insight on whether we should perform a joint search in DIANN with both sample types together (grey+white) or whether we should search separately given they are heterogeneous and would have distinct protein compositions?

**Comment by vdemichev** [comment]
Hi,

Most likely separate search will be better.

Best,
Vadim

**Reply by abadgerw** [reply]
Thank you! Just for my learning, what would you expect to be the difference? Would we have more false discoveries and/or more missing values with a joint search?

**Reply by vdemichev** [reply]
Just most likely slightly better ID numbers and slightly better quantification.
